test/e2e/gc_e2e_test.go

```diff
@@ -284,4 +284,101 @@ var _ = Describe("Garbage collection for dependent resources", func() {
 
 	})
 
+	Context("When installing a bundle with configmap and secret objects these objects should be deleted when the CSV is deleted", func() {
```

Comment:
the `should be deleted when CSV is deleted` could just be part of the `It` spec description. 
The way the Container block descriptions are written is :
`Describe` blocks to describe the individual behaviors of your code and `Context` blocks exercise those behaviors under different circumstances. You could additionally use `When` for better readability. 
For example:
```
[Describe] Subscriptions create required objects from Catalogs
  [Context] Given a Namespace with an OwnNamespace OperatorGroup
      [When] a CatalogSource is created with a bundle that contains prometheus objects
         [It] should install the operator successfully
             [By] creating a subscription to the CatalogSource
         [It] should have created the expected prometheus objects
```
Also, note that: `Describe/ Context / When` are functionally identical and only differ in their names.